client/packages/programs/src/JsonForms/common/JsonForm.tsx

```diff
@@ -28,7 +28,7 @@ import {
   dateTester,
   Date,
   arrayTester,
-  Array,
+  ArrayControl,
```

Comment:
I renamed this to avoid a name collision, as I wanted to use a built-in `Array` method, namely: `new Array(x).fill()`